### DISCUSSION TITLE
Why "cd $EXPORT_NAME" and git version

### DISCUSSION THREAD

**Discussion by knobliibrot** [abarichello/godot-ci#67] [Q&A] [answered]
First of all, big thanks to what you're doing here. It makes our lives so much easier.
It's the first time I work with Github Actions so please forgive me if I ask stupid things.

I just finished a build action for my game server. On the way I had two problems which could resolve. 

1. I still had my project at Godot version 3.2 so I also chose the respective Docker Image. But the Git version on the image was not recent one and so I couldn't build. For me it was no Problem cause my Project is small and I could with now effort update it. But for bigger projects it's may a problem. Is it may possible to automate that at all late docker images the git version get's updated from time to time?

2.  And I don't really understand why in the run stage (for Linux export) is "cd $EXPORT_NAME" used. It'caused an error for me, which I could resolve with deleting it and adjust the export path.

**Comment by OverloadedOrama** [comment, marked as answer]
`cd $EXPORT_NAME` is required for the example project because the project files are under the "test-project" directory. If in your project the files (which I assume are at least `project.godot` and `export_presets.cfg`) are in the root folder of the repository, then the change directory (cd) line is not required.
